app/views/topical_events/show.html.erb

```diff
@@ -4,31 +4,29 @@
 
 <%= content_tag_for(:div, @topical_event, class: "classification #{@topical_event.class.name.underscore}") do %>
 
-  <header class="block headings-block">
-    <div class="inner-block floated-children">
-      <%= render partial: 'shared/heading',
-                locals: { heading: "#{h(@topical_event.name)}#{' <span class="archived">(Archived)</span>' if @topical_event.archived?}".html_safe,
-                          big: true, extra: true } %>
-      <% if @topical_event.logo_url.present? %>
-        <div class="heading-extra topical-event-logo">
-          <div class="heading-inner">
-            <%= image_tag(@topical_event.logo_url(:s300), alt: @topical_event.logo_alt_text) if @topical_event.logo_url %>
-          </div>
+  <header class="headings-block">
+    <div class="inner-block">
```

Comment:
This is part of whitehall's gross custom grid system. Is it still necessary? Could it be replaced with `govuk-width-container` or removed altogether?

Reply:
We still need these until the main container with `class="whitehall-content"` will be replaced with `class="govuk-width-container"`. It's sort of a hybrid approach that would help us transition to the new layout system, otherwise it will require all layouts to be changed at once.

Reply:
Makes sense, bloody whitehall. Resolvable as well.